test/functional/lib/tests.cpp

```diff
@@ -4483,9 +4483,22 @@ AttachEbpfXdpProgram(
     )
 {
     unique_bpf_object BpfObject;
+    UINT32 RetryCount = 20;
+    HRESULT Result;
 
-    TEST_HRESULT(TryAttachEbpfXdpProgram(
-        BpfObject, If, BpfRelativeFileName, BpfProgramName, AttachFlags));
+    // TODO: https://github.com/microsoft/ebpf-for-windows/issues/2133
+    // Workaround till the above issue is fixed (and eBPF returns E_BUSY):
+    // Try a few times to load and attach the program with a sleep in between.
+    for (UINT32 i = 0; i < RetryCount; i++) {
+        Result = TryAttachEbpfXdpProgram(
+            BpfObject, If, BpfRelativeFileName, BpfProgramName, AttachFlags);
+        if (Result == S_OK) {
+            break;
+        }
+        // Sleep for 100ms between retries.
+        Sleep(100);
```

Comment:
Recommend using POLL_INTERVAL_MS here, though it shouldn't matter too much. You may also want to restructure the loop to use a watchog and retry until TEST_TIMEOUT_ASYNC_MS elapses.

Reply:
Fixed.